bin/offline-cluster.sh

```diff
@@ -9,36 +9,60 @@ set -x
 
 ls $ANSIBLE_DIR/inventory/offline
 
+if [ -f "$ANSIBLE_DIR/inventory/offline/hosts.ini" ]
+then
+        INVENTORY_FILE="$ANSIBLE_DIR/inventory/offline/hosts.ini"
+else
+        if [ -f "$ANSIBLE_DIR/inventory/offline/inventory.yml" ]
+        then
+                INVENTORY_FILE="$ANSIBLE_DIR/inventory/offline/inventory.yml"
+        else
+                {
+                        echo "no inventory file in ansible/inventory/offline/. please supply an inventory.yml or hosts.ini."
```

Comment:
```suggestion
                        echo "No inventory file in "$ANSIBLE_DIR/inventory/offline/. Please supply an inventory.yml or hosts.ini."
```
This suggestion contains two things:
1. The first word in a sentence starts with an uppercase letter.
2. We can be precise and refer to `"$ANSIBLE_DIR`